opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -412,8 +414,16 @@ namespace Opm {
         }
     }
 
+    std::vector< int > EclipseState::getRegions() const {
```

Comment:
This should in my opinion take  a `const std::string& regionName` input. The `Equil` initiialisation in opm-core probably already has a "similar" implementation based on `EQLNUM`